.eslintrc.js

```diff
@@ -53,6 +53,14 @@ module.exports = {
 					'@typescript-eslint/no-var-requires': 'off',
 					// REST API objects include underscores
 					'@typescript-eslint/camelcase': 'off',
+					'valid-jsdoc': [
+						2,
+						{
+							requireParamType: false,
+							requireReturn: false,
```

Comment:
This was removed to align with the rule configuration we currently use:

https://github.com/Automattic/wp-calypso/blob/202689d48e0345bf1ad1ab87fbf2a3904ca9f216/packages/eslint-config-wpcalypso/index.js#L113